torch/utils/data/sampler.py

```diff
@@ -104,10 +104,14 @@ def num_samples(self):
 
```

Comment:
Can you make modifications to the docstring in line 74 to elaborate on the changes?